### PR TITLE
Added precommit hooks for prettier, it will automatically run before committing

### PR DESCRIPTION
**Pull request by mt-empty**
# 
## Proposed changes

added precommit hooks for prettier, it will automatically run before committing.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [x] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,10 +24,23 @@
...
     "axios": "^0.21.1",
     "express": "^4.17.1"
+  },
+  "lint-staged": {
+    "**/*": "prettier --write --ignore-unknown",
+    "**/*.js": "standard --fix",
+    "**/*.ts": "ts-standard --fix"
+  },
+  "prettier": {
+    "printWidth": 100
```
Comment (reteppeter) [inline package.json:44]: Is it possible to use “tabWidth”: 4

**Review by reteppeter — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mt-empty) [reply package.json:44]: JSON files look horrible with a `"tabWidth”: 4`, so the default behaviour is fine.

Comment (reteppeter) [reply package.json:44]: I disagree, and find a tab width of 2 completely unreadable, but I was waiting for other peoples opinions.
However, if we use "useTabs: true" then both of use could be happy (this only affects indentation, spaces are still used for alignment).

**Review by reteppeter — CHANGES_REQUESTED** [review]
See if "useTabs: true" is possible.

**Review by beining00 — APPROVED** [review]

**Review by YinghaoMa — APPROVED** [review]
good

File: package.json (hunk 1 above)
Comment (mt-empty) [reply package.json]: Ok, I'm convinced, made the changes required.

**Review by reteppeter — APPROVED** [review]

**Pull request merged**